python/cuml/test/test_random_forest.py

```diff
@@ -550,3 +552,47 @@ def test_rf_regression_sparse(datatype, split_algo, mode, column_info,
         sk_r2 = r2_score(y_test, sk_predict, convert_dtype=datatype)
         assert fil_r2 >= (sk_r2 - 0.07)
     assert fil_r2 >= (cu_r2 - 0.02)
+
+
+@pytest.mark.parametrize('datatype', [np.float32])
+@pytest.mark.parametrize('column_info', [unit_param([20, 10]),
```

Comment:
Not sure if we need a lot of settings here. Maybe run it with 2000 rows and 20 columns for normal testing, and 1M rows for stress testing.